src/send_packets.c

```diff
@@ -498,11 +498,13 @@ send_packets(tcpreplay_t *ctx, pcap_t *pcap, int idx)
     bool now_is_now;
 
     ctx->skip_packets = 0;
-    start_us = TIMEVAL_TO_MICROSEC(&ctx->stats.start_time);
+    ctx->first_time = true;
+    gettimeofday(&now, NULL);
```

Comment:
Hi @fklassen, it's not every packet that gets a timestamp here, this is outside the "while get_next_packet", so every iteration will get an initial timestamp. In small traces in a loop, I guess this is quite frequent though. Would protecting it with a `!top_speed` make the diff merge-able?

To reproduce: `tcpreplay -i if0 trace1.pcap trace2.pcap` where trace1 != trace2. In my case they were traces split from a bigger pcap file, but I don't think it matters. For trace2 `first_pkt_ts` will be cleared and `start_us` referring to the time trace1 started, so it would try to sleep for a huge amount of time.